random-ballerina/build.gradle

```diff
@@ -45,197 +54,28 @@ def stripBallerinaExtensionVersion(String extVersion) {
     }
 }
 
-configurations {
-    jbalTools
-}
-
-dependencies {
-    compile group: 'org.ballerinalang', name: 'ballerina-runtime', version: "${ballerinaLangVersion}"
-    jbalTools ("org.ballerinalang:jballerina-tools:${ballerinaLangVersion}") {
-        transitive = false
-    }
-    compile project(":${packageName}-native")
-}
-
-clean {
-    delete "$project.projectDir/target"
-}
-
-jar {
-    manifest {
-        attributes('Implementation-Title': project.name,
-                'Implementation-Version': project.version)
-    }
-}
+apply plugin: 'io.ballerina.plugin'
 
-task unpackJballerinaTools(type: Copy) {
-    configurations.jbalTools.resolvedConfiguration.resolvedArtifacts.each { artifact ->
-        from zipTree(artifact.getFile())
-        into new File("${buildDir}/target/extracted-distributions", "jballerina-tools-zip")
-    }
+ballerina{
+    module = packageName
+    langVersion = ballerinaLangVersion
+    testCoverageParam = "--code-coverage --jacoco-xml --includes=org.ballerinalang.stdlib.random.*:ballerina.random.*"
 }
 
-task updateTomlVersions {
+tasks.create("updateTomlVersions"){
```

Comment:
Any reason to use tasks.create for this module?

Reply:
No,  I have changed it when testing the plugin,  It does not make any difference. Will change it